### PR TITLE
Opentelemetry tracing implementation 

### PR DESCRIPTION
**Pull request by vldm**

### CODE REVIEW

File: cli/src/clap_app.rs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 use crate::{
     cli::*, cluster_query::*, evm::EvmSubCommands, feature::*, inflation::*, nonce::*, program::*,
-    stake::*, validator_info::*, vote::*
```
Comment (gBaGu) [inline cli/src/clap_app.rs]: Do we need to bring this changes into this PR?

File: client/src/rpc_client.rs [hunk 2]
```diff
@@ -1817,10 +1817,13 @@ impl RpcClient {
...
     }
 
-    pub fn get_evm_transaction_receipt(&self, hash: &evm_state::H256) -> ClientResult<Option<evm_rpc::RPCReceipt>> {
+    pub fn get_evm_transaction_receipt(
+        &self,
+        hash: &evm_state::H256,
+    ) -> ClientResult<Option<evm_rpc::RPCReceipt>> {
         self.send::<Option<evm_rpc::RPCReceipt>>(
             RpcRequest::EthGetTransactionReceipt,
-            json!([evm_rpc::Hex(*hash)])
```
Comment (gBaGu) [inline client/src/rpc_client.rs]: Do we need to bring this changes into this PR?

File: core/src/snapshot_packager_service.rs [hunk 3]
```diff
@@ -157,9 +157,7 @@ mod tests {
             fs::hard_link(&snapshots_path, &link_path).unwrap();
         }
 
-        let GenesisConfigInfo {
-            genesis_config, ..
-        } = create_genesis_config(10_000);
```
Comment (gBaGu) [inline core/src/snapshot_packager_service.rs]: Do we need to bring this changes into this PR?

File: evm-utils/evm-bridge/src/pool/listener.rs [hunk 4]
```diff
@@ -6,6 +6,7 @@ use txpool::Listener;
 
 use super::PooledTransaction;
 
+#[derive(Debug)]
```
Comment (gBaGu) [inline evm-utils/evm-bridge/src/pool/listener.rs:10]: Do we need to bring this changes into this PR?

File: evm-utils/evm-bridge/src/sol_proxy.rs [hunk 5]
```diff
@@ -281,7 +281,12 @@ impl rpc::rpc_full::Full for FullRpcSolProxy {
         address_strs: Vec<String>,
         config: Option<RpcEpochConfig>,
     ) -> BoxFuture<Result<Vec<Option<RpcInflationReward>>>> {
-        Box::pin(ready(proxy_sol_rpc!(meta.rpc_client, GetInflationReward, address_strs, config)))
```
Comment (gBaGu) [inline evm-utils/evm-bridge/src/sol_proxy.rs]: The same question for this file)

File: evm-utils/evm-bridge/src/pool/listener.rs (hunk 4 above)
Comment (vldm) [reply evm-utils/evm-bridge/src/pool/listener.rs:10]: Debug is generally good thing to have for all structures.

**Review by gBaGu — APPROVED** [review]

**Pull request merged**